hibernate-reactive-core/src/main/java/org/hibernate/reactive/bulk/impl/ReactiveBulkIdStrategy.java

```diff
@@ -209,6 +212,26 @@ public TableBasedUpdateHandlerImpl(SessionFactoryImplementor factory, HqlSqlWalk
 			this.parameterSpecifications = parameterSpecifications.toArray( new ParameterSpecification[0][] );
 		}
 
+		@Override
+		protected String generateIdInsertSelect(
+				String tableAlias, IdTableInfo idTableInfo, ProcessedWhereClause whereClause) {
+			String sql = super.generateIdInsertSelect( tableAlias, idTableInfo, whereClause );
+			return Parameters.process( sql );
```

Comment:
Why so many little local variables named `sql`?

Reply:
I find it easier for debugging